server/services/database/documents.js

```diff
@@ -18,8 +18,30 @@ const query =
       on u.id = d.owner_id";
 
 const getAllDocuments = () => {
-  const sqlQuery = query + ";";
-
+  const sqlQuery = `select\
+  d.id, d.format, d.name,\
+    d.status, d.submission_date, d.due_date,\
+    lf.name as from_language_name,\
+    lt.name as to_language_name,\
+    u.name as owner_name,\
+    trnsl.start_date, trnsl.translator_name\
+  from documents as d\
+  inner join languages as lf\
+  on  lf.code = d.from_language_code\
+  inner join languages as lt\
+  on lt.code = d.to_language_code\
+  inner join users as u\
+  on u.id = d.owner_id\
+  left join(\
+    select\
+           t.document_id,\
+    t.start_date,\
+    tu.name as translator_name\
+    from translations as t\
+    inner join users as tu\
+        on tu.id = t.user_id\
+  ) trnsl on trnsl.document_id = d.id\
+    ;`;
```

Comment:
Does it not make sense to also read (and display) the processing state in other queries, such as `getUserDocuments` (wouldn't the user want to see who is translating their document)? That way the SQL can also we shared as it was before which would make the code simpler.

Reply:
I agree, it should also be returned by getUserDocuments otherwise in the dashboard table, for users, it is displayed "Processing by" (and just empty after).